Resources.html

```diff
@@ -81,25 +101,65 @@
         </div>
     </nav>
   
-    <div class="container-fluid text-center" style="height: 100%; padding-top: 50px; padding-bottom: 60px;">    
-        <div class="row content" style="height: 100%;">
-    <div class="col-sm-2 sidenav">
-      <p><a href="#">Link</a></p>
-      <p><a href="#">Link</a></p>
-      <p><a href="#">Link</a></p>
-    </div>
-    <div class="col-sm-10 text-left"> 
-      <h1>Resources</h1>
-      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+<div class="container-fluid text-center" style="height: 100%; padding-top: 50px; padding-bottom: 60px;">    
+    <div class="col-sm-10 text-center"></div>
+      <h1>Getting Started</h1>
+      <h4>Cybersecurity encompasses so many areas that it's hard to know where to start. Here are some of our recommendations for learning and practicing the basics when it comes to hacking the web and beyond!</h4>
       <hr>
-      <h3>Test</h3>
-      <p>Lorem ipsum...</p>
-    </div>
-  </div>
+      <div class="col-sm-6 col-md-6 col-lg-6 text-left">
+          <div class="row">
+            <div class="div-highlight">
+            <h3>Where to practice</h3>
+            </div>
+            <a href="https://overthewire.org/wargames/bandit/" target="_blank"><b>Over the Wire</b></a><p>A free series of wargames starting with "Bandit" that help you learn and practice security concepts in successive stages that build on each other.</p>
```

Comment:
A comment which applies to all of these descriptions/sections, change the `href` from being the title/name of the tool to the entire div, as such when a user clicks on that div for that particular resource, that page is opened for them.